src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-pg-scrub-modal/osd-pg-scrub-modal.component.html

```diff
@@ -0,0 +1,50 @@
+<cd-modal [modalRef]="bsModalRef">
+  <h4 i18n="form title|Example: Create Pool@@formTitle"
```

Comment:
`cd-modal` component will put the title inside a h4 with the proper classes:
```suggestion
  <ng-container i18n="form title|Example: Create Pool@@formTitle"
```